schema/material.json

```diff
@@ -29,6 +29,10 @@
             "description": "object defining the lattice cell size and shape",
             "$ref": "properties_directory/derived_properties.json"
         },
+        "electronicConfiguration": {
+            "description": "object defining the number of alpha and beta electrons (via charge and multiplicity)",
```

Comment:
(1) It is not entirely clear what alpha and beta electrons are - should we put a link to the explanation?

(2) There is a way to associate "proto_properties" with materials in our platform (the atomic_constraints, for example, is one such property). The electronic configuration seems more like such property and seems to be more related to the workflow than the structural parameters. Should we just leave this key out for now?